samples/minimal-samples/powerflex.yaml

```diff
@@ -14,7 +14,17 @@ spec:
       # enable: Enable/Disable csm-authorization
       enabled: false
       # For Auth 2.0, use v2.0.0 as configVersion
-      configVersion: v1.11.0
+      configVersion: v1.12.0
+      components:
+        - name: karavi-authorization-proxy
+          image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
```

Comment:
Do we really need this to be explicitly listed here? This is the default image

Reply:
Addressed